app/models/product.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class Product < ApplicationRecord
+  has_one_attached :image
+
+  validates :image, content_type: {
+    in: %w[image/png image/jpg image/jpeg],
+    message: 'This format is not allowed. Allowed formats are jpg, jpeg, png',
+  }, size: {
+    less_than: 100.megabytes,
+    message: 'File size is too big. Max allowed size is 100 Mb.',
+  }
+
+  validates_presence_of :name, message: 'This field is required'
+  validates_length_of :name, minimum: 2, maximum: 500, message: 'Min characters for this field is 2. Max characters for this field is 500'
```

Comment:
use `validates` syntax, validates_presence_of, validates_length_of looks more complex, and follow our rails style guides

Reply:
I don't know if it is possible, but try to move all validation messages to locales file, this will help to keep models clear

Reply:
https://codedecoder.wordpress.com/2013/09/11/customize-error-message-with-locales-rails/